runtime/libtock_layout.ld

```diff
@@ -91,6 +92,7 @@ SECTIONS {
      * https://riscv.org/wp-content/uploads/2015/01/riscv-calling.pdf
      */
     .stack (NOLOAD) : {
+        _sram_origin = .;
```

Comment:
Where is `_sram_origin` used?

Reply:
`_sram_origin` is used by elf2tab: https://github.com/tock/elf2tab/blob/008cd9e87876415f0bad62648fceb279ffa7a93f/src/main.rs#L301

Reply:
I didn't know that, so I definitely think it's worth a comment :-)

Reply:
Added.